projects/step-core/src/lib/modules/table/components/table/table.component.scss

```diff
@@ -7,12 +11,12 @@ table {
   width: 100%;
 }
 
-tr.mat-row:hover {
-  background: var(--darken-background-10);
+::ng-deep .mat-form-field-infix {
```

Comment:
Did you check if it is possible to implement this without ::ng-deep? If not, at least add :host before it so it does not affect the whole app.

Reply:
I tried everything I could think of before using `::ng-deep`. Putting it into `:host` makes perfectly sense, just did it :+1:

Reply:
The gaps in the lines are reason why I created an extra PR: https://github.com/exense/step-frontend/pull/133 so that you can decide what you like more.

Actually, **this** PR (132) does not produce any gaps.

Reply:
the only drawback is that I could not achieve to add a padding only to the content of the cells. so when the table is squeezed, there is no space between cells (see screenshot above). so either no space between cells, or the look of #133 .

Reply:
Actually this PR (#132) has the issue with the gap, I did not look at the other one. The gap is a no-go, but we also need to have some padding. 

Just a quick look for solutions:
remove 
`border-spacing: 10px 0!important;` for tr
add 
```
td.mat-cell {
    padding: 0 8px;
}
```

Probably you would have to be a bit more specific which cells should get padding (maybe not the checkbox) and maybe have to set the paddign according to viewport.

Second solution is to simply remove `border-collapse: separate!important;` for the table. I don't see why we have this set but maybe check if there is any problem with removing it.